Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -153,6 +153,68 @@ final class TensorAutoDiffTests: XCTestCase {
         XCTAssertEqual(varianceGradAlongAxes(input), expected)
     }
 
+    func testMin() {
+        // The expected gradient values were computed using the following TensorFlow 2.0 Beta1
+        // Python code with respective `a` and `b` tensors :
+        // ```
+        // with tf.GradientTape() as t:
+        //     t.watch([a, b])
+        //     y = tf.math.reduce_sum(tf.minimum(a, b))
+        // print(t.gradient(y, [a, b]))
+        // ```
+        func f(a: Tensor<Float>, b: Tensor<Float>) -> Tensor<Float> { min(a, b).sum() }
+        var a = Tensor<Float>([4, 5, 3])
```

Comment:
Could you please use `let` variables instead of `var`?
You can use `do { ... }` blocks to avoid variable redeclaration issues.
```swift
do {
    let a = Tensor<Float>([4, 5, 3])
    ...
}

do {
    let a = Tensor<Float>([[3.0, -2.0], [0.3, 10.0]])
    ...
}
```

Reply:
Done in https://github.com/tensorflow/swift-apis/pull/480/commits/7eba493331c07f204d487e4869af137516d496b0.